tests/testthat/helper-cens-churn.R

```diff
@@ -0,0 +1,33 @@
+
+make_churn_cens_objects <- function(x) {
+  suppressPackageStartupMessages(require("tidymodels"))
+  suppressPackageStartupMessages(require("censored"))
+
+  data("mlc_churn")
+
+  mlc_churn <-
+    mlc_churn %>%
+    mutate(
+      churned = ifelse(churn == "yes", 1, 0),
+      event_time = Surv(account_length, churned)
```

Comment:
```suggestion
      event_time = survival::Surv(account_length, churned)
```
I see the `require("censored")` a couple of lines earlier but for some reason I got an error about `Surv()` not being found.